src/quo2/components/inputs/input/view.cljs

```diff
@@ -136,7 +135,8 @@
 (defn input
   "This input supports the following properties:
   - :type - Can be `:text`(default) or `:password`.
-  - :variant - :light(default), :light-blur, :dark or :dark-blur.
+  - :blur - Boolean to set the blur color variant.
```

Comment:
`blur?` ??

Reply:
@J-Son89 

I usually don't add `?` at the end of the prop names on the component's API, that's because I try to be consistent with react components. Examples of react properties expeciting a bool are `editable` (`disabled`  in html), `auto-correct`, `auto-focus`, `multiline`, etc.

At the end, even if we add `?` to all Clojure boolean props, we will be passing booleans to props ending with and without `?` due to existing React components params.

What do you think?
Have you discussed this before?

Reply:
I've no strong opinions, but maybe other devs on the team do 🤔

Reply:
Same here, I'll ask on Status

Reply:
related thread: https://discord.com/channels/624205794384281629/1024249352060928010/1024253324406554624

Reply:
Ready!